src/com/palmergames/bukkit/towny/permissions/PermissionNodes.java

```diff
@@ -378,6 +378,7 @@
 	TOWNY_CHAT_SPY("towny.chat.spy"),
 	
 	TOWNY_OUTLAW_JAILER("towny.outlaw.jailer"),
+	TOWNY_OUTLAW_TPBYPASS("towny.outlaw.tpbypass"),

```

Comment:
Replace this with `TOWNY_ADMIN_OUTLAW_TELEPORT_BYPASS("towny.admin.outlaw.teleportbypass"),` added to line 10, below the admin nation zone node.

Reply:
Done, but with teleport_bypass to be consistent with nation_zone.